cvr/src/main/java/org/cbioportal/cmo/pipelines/cvr/BatchConfiguration.java

```diff
@@ -772,4 +784,76 @@ public JobExecutionDecider svFusionsStepExecutionDecider() {
     public JobExecutionDecider segStepExecutionDecider() {
         return decideStepExecutionByDatatypeForStudyId("seg");
     }
+
+    // general spring batch configuration
+    @Value("org/springframework/batch/core/schema-drop-sqlite.sql")
+    private Resource dropRepositoryTables;
+
+    @Value("org/springframework/batch/core/schema-sqlite.sql")
+    private Resource dataRepositorySchema;
```

Comment:
all of the changes in the BatchConfiguration classes are the same. 

These are required beans that need to be explicitly defined in the newer version of Spring Batch that we're using now

Reply:
✅